### DISCUSSION TITLE
Open-Shell 4.4.190 won't install, Windows 10 Home

### DISCUSSION THREAD

**Discussion by gitedgr** [Open-Shell/Open-Shell-Menu#1615] [Q&A]
Today I downloaded OpenShellSetup_4_4_190.exe and tried to install it, but "Windows Security" completely blocks it from installing.  It reports: Trojan:Win32/Bearfoos.A!ml
1. Is this a false positive?
2. If so, how do I get around Windows Security?

**Comment by ge0rdi** [comment]
> Is this a false positive?

Yes, it is.
You should report it to MS.

> If so, how do I get around Windows Security?

I guess you should be able to add some [exclusion](https://support.microsoft.com/en-us/windows/add-an-exclusion-to-windows-security-811816c0-4dfd-af4a-47e4-c301afe13b26#:~:text=Go%20to%20Start%20%3E%20Settings%20%3E%20Update,%2C%20file%20types%2C%20or%20process).

Or just disable Defender for a while?
(I'm sorry I don't use Windows Defender)

Or just wait a while?
These bogus detections get fixed usually after some time ...

**Reply by pykd3** [reply]
Virustotal isnt happy too... 

https://www.virustotal.com/gui/file/31e616f27fad0c9b14a22c02ded2dc524411abf53d72c4136f22db1be0156865

I think the most weight on that false detection is the Startmenu.exe

https://www.virustotal.com/gui/file/08f505b325a67b61eb997cd45d61fb04851b6e6477110739a7cfc1ef5d290fae

**Comment by gitedgr** [comment]
For what it's worth, today (Jul 27/23) I tried again to install the current version of Open-Shell, and it installed (on 3 separate PCs) with no questions asked.  Maybe someone reported it as a false positive to MS, and they are no longer blocking it…
